graspologic/plot/plot.py

```diff
@@ -341,6 +341,42 @@ def heatmap(
     return plot
 
 
+def binary_heatmap(X, colors=["white", "black"], **kwargs):
+    """
+    Plots an unweighted graph as a black-and-white matrix
+    with a binary colorbar.
```

Comment:
Added in this PR because I did this while I was making the CASE tutorial. Can refactor into a separate PR.